jetson-xavier-seeed-hat.coffee

```diff
@@ -0,0 +1,44 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BOARD_POWEROFF = 'Remove power from the Jetson Xavier NX Devkit SD-CARD and insert the freshly burnt sd-card.'
+BOARD_POWERON  = 'Power on the board.'
+
+module.exports =
+	version: 1
+	slug: 'jetson-xavier-seeed-hat'
```

Comment:
I suggest we include the keywords 'nx' and 'sd' in the slug, yocto machine and coffee file name so it's more clear that this is not a Xavier AGX.